RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+*   **Accordion, clipboard copy, data list, dropdown, expandable, nav, options menu, select, table, wizard**: Added expanded/collapsed arrow rotation in components ([#1932](https://github.com/patternfly/patternfly-next/pull/1932))
+*   **Accordion:** Add variation of accordion that uses divs and headings instead of definition list [(#1990](https://github.com/patternfly/patternfly-next/pull/1990))
+*   **Alert group:** Renamed example and reworded docs [(#1930](https://github.com/patternfly/patternfly-next/pull/1930))
+*   **App launcher**
+    *   Used `--pf-global--icon--FontSize--lg` for the icon size. Reduced height of expanded demo  ([#1935](https://github.com/patternfly/patternfly-next/pull/1935))
+    *   Added support for sections/icons/divider ([#1916](https://github.com/patternfly/patternfly-next/pull/1916))
+*   **Background image:** Reduce size of background image ([#1936](https://github.com/patternfly/patternfly-next/pull/1936))
+*   **Card:** Added compact variation ([#1975](https://github.com/patternfly/patternfly-next/pull/1975))
+*   **Charts:** Added remaining variables ([#1863](https://github.com/patternfly/patternfly-next/pull/1863))
+*   **Copy to Clipboard:** fixed focus ring and add content editable to expanded ([#1896](https://github.com/patternfly/patternfly-next/pull/1896))
+*   **Datalist**: Added an example using a heading in the primary content section. ([#1870](https://github.com/patternfly/patternfly-next/pull/1870))
+*   **Dropdown:** Increased height of examples that overflow ([#1965](https://github.com/patternfly/patternfly-next/pull/1965))
+*   **Expandable:** Added type to the button ([#1982](https://github.com/patternfly/patternfly-next/pull/1982))
+*   **Form:** Added element for form label text, make it bold ([#1952](https://github.com/patternfly/patternfly-next/pull/1952))
+*   **Form, login, wizard:** Fixed order of variables which was causing the IE conversion script to generate undefined. Also removed a login variable that was not needed. This was also causing the IE scripts to generate undefined. ([#1871](https://github.com/patternfly/patternfly-next/pull/1871)).
+*   **Nav:** Changed breakpoint to fix bug ([#1918](https://github.com/patternfly/patternfly-next/pull/1918))
+*   **Options menu:** Added disabled variation ([#1973](https://github.com/patternfly/patternfly-next/pull/1973))
+*   **Progress:** Changed the font weight on sm variation ([#1974](https://github.com/patternfly/patternfly-next/pull/1974))
+*   **Table**
+    *   Added an extra breakpoint. In the data table sortable demo the table needed a larger breakpoint, so that the rows don't overflow outside of their container ([#1871](https://github.com/patternfly/patternfly-next/pull/1880))
```

Comment:
```suggestion
    *   Added an extra breakpoint. In the data table sortable demo the table needed a larger breakpoint, so that the rows don't overflow outside of their container ([#1880](https://github.com/patternfly/patternfly-next/pull/1880))
```